tools/mo/unit_tests/moc_tf_fe/conversion_basic_models.py

```diff
@@ -43,14 +43,8 @@ def base_args_config():
     args.scale_values = None
     args.output_dir = os.getcwd()
     args.freeze_placeholder_with_value = None
-    args.tensorflow_use_custom_operations_config = None
```

Comment:
here we remove tensorflow_use_custom_operations_config option. Should we, if we are not going to remove it in this PR?

Reply:
Answered above. Let's continue there